k9mail/src/main/java/com/fsck/k9/notification/NotificationController.java

```diff
@@ -2,15 +2,26 @@
 
 
 import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.PorterDuff;
+import android.graphics.PorterDuffXfermode;
+import android.graphics.Rect;
+import android.graphics.RectF;
```

Comment:
These imports are no longer needed here (remnant of an earlier implementation which I had tried).